src/rootPages/Designer/ui_work_interface_workspace_details_properties.js

```diff
@@ -62,6 +60,14 @@ export default function (AB) {
 
             // Timed saves ... once every 3s of no changes
             let view = this.CurrentView;
+            this.pendingSave = true;
+
+            // Add a beforeunload event listener to save changes if the user leaves
+            if (!this.unloadListener) {
+               this.unloadListener = webix.event(window, "beforeunload", () => {
```

Comment:
It might be good if we use `this.AB.Webix` instead of the global `webix` for easy to mock in unit testing.